### PR TITLE
frap figure script py3

### PR DESCRIPTION
**Pull request by will-moore**
Py3 fixes - a little overdue.
This script is uploaded to the workshop server.

### CODE REVIEW

**Comment by jburel** [comment]
i have started to filter this repo and move those scripts to omero-guide-python

**Comment by jburel** [comment]
few flake8 issues

We can also probably drop python 2.7 from travis

**Comment by jburel** [comment]
@will-moore I fixed travis failure

**Comment by jburel** [comment]
now that we have dedicated location that we could consider moving that script to omero-guide-figure

File: practical/python/server/simple_frap_with_figure.py [hunk 1]
```diff
@@ -114,10 +114,10 @@ def create_figure_file(conn, figure_json):
...
     conn.SERVICE_OPTS.setOmeroGroup(gid)
 
-    json_string = json.dumps(figure_json)
-    file_size = len(json_string)
-    f = StringIO()
-    json.dump(figure_json, f)
+    json_bytes = json.dumps(figure_json).encode('utf-8')
+    file_size = len(json_bytes)
+    f = BytesIO()
+    f.write(json_bytes)
```
Comment (jburel) [inline practical/python/server/simple_frap_with_figure.py:120]: This will need to be closed

**Comment by jburel** [comment]
Year in header should also be updated

**Comment by jburel** [comment]
Screenshot 2020-05-22 at 08 59 45

**Comment by will-moore** [comment]
OK, I'll move this script to https://github.com/ome/omero-guide-figure.
Is it worth merging here too?

**Comment by will-moore** [comment]
Actually, this script is not used in the omero-guide-figure but in the omero-guide-python.
So I'll move it there.

**Comment by jburel** [comment]
To keep track of the history it will be better to merge it here, then filter

**Comment by jburel** [comment]
Have you uploaded the latest version of the script for a final check?

**Comment by will-moore** [comment]
Uploaded to workshop server just now.

**Comment by jburel** [comment]
Merging this PR, I will filter and open a PR against omero-guide-python

**Pull request merged**